v0/util/index.js

```diff
@@ -1151,6 +1151,43 @@ function isCdkDestination(event) {
   );
 }
 
+/**
+ * This function helps to remove any undefined value from dynamicForm, dynamicCustomForm
+ * and dynamicSelectForm array.
+ *
+ * example:
+ * input: "eventsToEvents": [
+                    {
+                        "from": "spin_result",
+                        "to": "Schedule"
+                    },
+                    {
+                        "to": "Schedule"
+                    }
+                ]
+ * output: "eventsToEvents": [
+                    {
+                        "from": "spin_result",
+                        "to": "Schedule"
+                    }
+                ]
+
+ * @param {*} eventArray -> This is the dynamicForm/dynamicCustomForm/dynamicSelectForm element.
+ * @param {*} keyLeft -> This is the name of the leftKey, in general it is 'from'.
+ * @param {*} keyRight -> This is the name of the rightKey, in general it is 'to'.
+ * @returns
+ */
+function validateDynamicFormAttribute(attributeArray, keyLeft, keyRight) {
+  let res = [];
+  res = attributeArray.filter(element => {
+    return (
+      (element[`${keyLeft}`] || element[`${keyLeft}`] === "") &&
+      (element[`${keyRight}`] || element[`${keyRight}`] === "")
```

Comment:
Do we need the string conversions here? If they were not strings in the first place, converting them to strings wouldn't help.

Reply:
No removing the conversions.